pdr_backend/lake/gql_data_factory.py

```diff
@@ -69,10 +69,34 @@ def __init__(self, ppss: PPSS):
             }
         }
 
-    def _prepare_temp_table(self, dataclass: Type[LakeMapper], st_ut, fin_ut):
+    @enforce_types
+    def _calc_start_ut(self, table: Table) -> UnixTimeMs:
```

Comment:
I think this can also be a property of the table, e.g. `etl_start_ut`

Reply:
same for what involves end_ut

Reply:
please read my comments/feedback

we should not add this logic to table...